### PR TITLE
Fix/quick order uom issue

### PR DESCRIPTION
**Pull request by devinkulanjith**
#### What does this PR do? \*
* Add UoM filed in quick order search result

#### How to test it? \*
[Workspace](https://productsearch--sbdsefprod.myvtex.com/quickorder?__bindingAddress=catalog2.sef.ao.sbd-prod.com/en-US/catalog-global)

### CODE REVIEW

File: react/AutocompleteBlock.tsx [hunk 1]
```diff
@@ -407,28 +457,20 @@ const AutocompleteBlock: FunctionComponent<any & WrappedComponentProps> = ({
...
                     className={`flex flex-column w-40 ph5-l ph2 p fl ${handles.inputQuantity}`}
                   >
-                    <Input
+                    <NumericStepper
+                      size={'small'}
                       value={quantitySelected}
-                      size="small"
-                      type="number"
-                      step={unitMultiplier}
+                      minValue={1}
```
Comment (Jayendra88) [inline react/AutocompleteBlock.tsx:463]: this should be moq

File: react/AutocompleteBlock.tsx [hunk 2]
```diff
@@ -407,28 +457,20 @@ const AutocompleteBlock: FunctionComponent<any & WrappedComponentProps> = ({
...
+                      maxValue={10000000}
```
Comment (Jayendra88) [inline react/AutocompleteBlock.tsx:464]: we should put the same values in product summary quantity selector

**Review by Jayendra88 — CHANGES_REQUESTED** [review]

File: react/AutocompleteBlock.tsx (hunk 1 above)
Comment (devinkulanjith) [reply react/AutocompleteBlock.tsx:463]: In sef-quantity-selector same logic is used , minvalue is taken as 1

File: react/AutocompleteBlock.tsx (hunk 2 above)
Comment (devinkulanjith) [reply react/AutocompleteBlock.tsx:464]: In sef-quantity-selector same logic is used , minvalue is taken as 1

**Pull request merged**